READMEs/fixed-rate-exchange-flow.md

```diff
@@ -99,7 +99,7 @@ print(f"config.provider_url = '{config.provider_url}'")
 #Alice's wallet
 import os
 from ocean_lib.web3_internal.wallet import Wallet
-alice_wallet = Wallet(ocean.web3, private_key=os.getenv('TEST_PRIVATE_KEY1'))
+alice_wallet = Wallet(ocean.web3, private_key=os.getenv('TEST_PRIVATE_KEY1'), block_confirmations=config.block_confirmations)
```

Comment:
Make a separate line for:

private_key = os.getenv(..)

Then make Wallet(..) briefer like the above readme.

Reply:
Done.